assets/src/dashboard/components/cardGridItem/storyCardPreview.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { __, sprintf } from '@web-stories-wp/i18n';
+import {
+  themeHelpers,
+  Button,
+  BUTTON_TYPES,
+  BUTTON_SIZES,
+  ThemeGlobals,
+} from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+import { resolveRoute } from '../../app/router';
+import { ActionLabel } from './types';
+
+const CardWrapper = styled.div`
+  height: 100%;
+  width: 100%;
+  position: relative;
+`;
+
+const Poster = styled.img`
+  display: block;
+  height: 100%;
+  width: 100%;
+  background: ${({ theme }) => theme.colors.gradient.placeholder};
+  object-fit: fill;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+`;
+
+const EditControls = styled.div`
+  width: 100%;
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  bottom: 0;
+  margin: 20px 0;
+  transition: opacity ease-in-out 300ms;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  opacity: 0;
+
+  ${({ theme }) => css`
+    &[${ThemeGlobals.FOCUS_VISIBLE_DATA_ATTRIBUTE}] {
+      ${themeHelpers.focusCSS(theme.colors.border.focus)};
+    }
+  `};
+`;
+
+const getActionAttributes = (targetAction) =>
+  typeof targetAction === 'string'
+    ? {
+        href: resolveRoute(targetAction),
+        isLink: true,
+        as: 'a',
+      }
+    : { onClick: targetAction };
+
+const StoryCardPreview = ({
```

Comment:
this became different enough from the shared `CardPreview` that I moved it to its own version.  Cleaned out everything that we have moved away from on the hover/focus of the card image too.